### PR TITLE
[Fix] base: Correct typos in res.country.state.csv

### PR DESCRIPTION
**Pull request by Row042**
Fixes two minor typos in the 'res.country.state.csv' file. The correction improves the accuracy of the data in this file.

Closes #131211

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Row042** [comment]
[FW][CLA] signature for Row042 #141642

**Comment by mart-e** [comment]
Hi,

This PR is for 14.0 but your CLA at #141458 was for 16.0. If you want to contribute to this version, you should add the signature in 14.0 as well (you can add it in this PR directly).

File: odoo/addons/base/data/res.country.state.csv [hunk 1]
```diff
@@ -297,8 +297,8 @@ state_eg_sin,eg,"North Sinai","SIN"
 state_eg_shg,eg,"Sohag","SHG"
 state_za_ec,za,"Eastern Cape","EC"
 state_za_fs,za,"Free State","FS"
-state_za_gt,za,"Gauteng","GT"
-state_za_nl,za,"KwaZulu-Natal","NL"
+state_za_gt,za,"Gauteng",GP
+state_za_nl,za,"KwaZulu-Natal",KZN
```
Comment (vava-odoo) [inline odoo/addons/base/data/res.country.state.csv:301]: Indeed, thanks for the change :+1: but it looks like you forgot the quotation mark
```suggestion
state_za_gt,za,"Gauteng","GP"
state_za_nl,za,"KwaZulu-Natal","KZN"
```

**Review by vava-odoo — APPROVED** [review]
Thanks! 
Could you also squash the last commit into the first one? (1 commit for the fix and 1 for the CLA)
And also prefix it with [FIX] instead of [REF].
Cheers

**Pull request closed without merging**